test/index.spec.js

```diff
@@ -154,8 +193,9 @@ function runTestWithConfig (testConfig, initialStoreState, label) {
       })
 
       it('should increment when action is dispatched to store', () => {
+        let expectedResult = store.getState().present + 1
         store.dispatch({type: 'INCREMENT'})
-        expect(store.getState()).to.deep.equal(incrementedState)
+        expect(store.getState().present).to.equal(expectedResult)
```

Comment:
If this becomes an issue in the long term we can re-initialize the store in-between tests with:
```js
  before('reinitialize store', () => {
    store = Redux.createStore(...)
  })
```